pkg/cli/admin/upgrade/status/mco/layered_node_state.go

```diff
@@ -256,15 +263,21 @@ func checkNodeReady(l *LayeredNodeState) error {
 		// - NodeDiskPressure condition status is ConditionFalse,
 		// - NodeNetworkUnavailable condition status is ConditionFalse.
 		if cond.Type == corev1.NodeReady && cond.Status != corev1.ConditionTrue {
-			l.ReasonOfUnavailability = fmt.Sprintf("node is reporting NotReady=%v with message=%q", cond.Status, cond.Message)
+			l.ReasonOfUnavailability = ReasonOfUnavailabilityNodeNotReady
+			l.MessageOfUnavailability = cond.Message
+			l.LastTransitionTimeOfUnavailability = cond.LastTransitionTime.Time
```

Comment:
One thing that I disliked about this structure (and that we can improve) is that we show only the first reason we hit, even if there are more (a node can be notready and have disk pressure, for example).

Can we accumulate the reasons (see my comment above on the `LayeredNodeState` changes) and report them all?

Reply:
Done